Rentit-frontend/src/pages/profile/profile.jsx

```diff
@@ -13,6 +14,7 @@ const Profile = () => {
 
   return (
     <>
+      <Subnavbar />
```

Comment:
Pls Follow the General Naming convection's throughout your web app. For more details, check this out -> https://climbtheladder.com/10-react-naming-conventions-best-practices/